hugo/layouts/partials/head.html

```diff
@@ -53,18 +64,30 @@
         crossorigin="anonymous">
     </script>
 
-    <!-- JQuery -->
-    <script src="https://code.jquery.com/jquery-3.7.1.min.js"
-        integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous">
-    </script>
+    <!-- Datatables (sometimes needs Jquery as dependancy) -->
+    {{ if or (eq .Title "Glossary") (eq .Title "Download") (eq .Title "Genome assembly") }}
+        <script src="https://code.jquery.com/jquery-3.7.1.min.js"
+            integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo=" crossorigin="anonymous">
+        </script>
+        <link href="/Datatables/datatables.min.css" rel="stylesheet">
```

Comment:
Point of curiosity: is there a reason why we don't rely on a CDN for Datatables ?  Or even JBrowse for that matter. I suddenly realize that I'm perhaps not quite aware of the tradeoffs.

Reply:
I think this [stackoverflow post](https://stackoverflow.com/questions/42591235/is-it-better-to-use-cdn-for-js-and-similar-resources-vs-local) was quite useful, so it seems like it could indeed be worth using a cdn instead. I've made a Jira task for this to investigate it (EB-200).

Reply:
I have on my side examined a bit more the CDN option for JBrowse, and in that case there might be good reasons to stick with our hosted assets. In short, the entry point exposed by the JBrowse CDN builds is lower level, we get more out of the box (e.g. plugin loading, relative URLs in `config.json`...) by using the jbrowse web releases. But I may start an experimental PR.